common/source/docs/common-auxiliary-functions.rst

```diff
@@ -170,6 +170,26 @@ Supported Features
 +----------------------+----------------------------+----------+---------+---------+
 |        77            | TAKEOFF mode               |          |    X    |         |
 +----------------------+----------------------------+----------+---------+---------+
+|        78            | RunCam Control             |    X     |    X    |    X    |
++----------------------+----------------------------+----------+---------+---------+
+|        79            | RunCam OSD Control         |    X     |    X    |    X    |
++----------------------+----------------------------+----------+---------+---------+
+|        80            | VISODOM Calibrate          |    X     |         |         |
```

Comment:
sorry to be picky but can we chcnage VISODOM to "VisualOdometry" or "VisOdom"?

Reply:
the meta data says Viso Align now
should match that